test/plugins/navigation/review/navigator/test.js

```diff
@@ -319,6 +325,48 @@ define([
 
                 ready();
             })
-            .render($container);
+            .render($container)
+            .update(sample.map, sample.context);
+        });
+
+    QUnit.module('Fizzy layout');
+
+    QUnit.test('close button exists', function(assert) {
+        const ready = assert.async();
+        const $container = $('#qunit-fixture');
+        const config = Object.assign({}, sample.config, { "reviewLayout": "fizzy"});
+
+        assert.expect(1);
+
+        navigatorFactory(config, sample.map, sample.context)
+        .on('update', function() {
+            const closeButton = $('.qti-navigator > .qti-navigator-label > .icon-close', $container);
+            assert.equal(
+                closeButton.length,
+                1,
+                'The panel contains close button'
+            );
+
+            ready();
+        })
+        .render($container)
+        .update(sample.map, sample.context);
+    });
+
+    QUnit.test('visual test', function(assert) {
+        const ready = assert.async();
+        const $container = $('#outside');
+        const config = Object.assign({}, sample.config, { "reviewLayout": "fizzy"});
+
+        assert.expect(1);
+
+        navigatorFactory(config, sample.map, sample.context)
+        .on('update', function() {
+            assert.ok(true);
```

Comment:
Optional suggestion:
Maybe we can add a check for informational item logic here - That `item.numberTest` is skipped for them?

And maybe a check for some other basic things, like there are N buttons (N= item count), but I'm not sure, since test for 'normal' layout also contains a very simple test-case like this.

Reply:
I will add test for new features. This exact code - only rendering the panel, for visual check, so such assertion is normal.

Reply:
Tests are added

Reply:
I've rechecked - all is good